zproject/settings.py

```diff
@@ -125,6 +125,9 @@ def get_secret(key: str) -> None:
     # Google auth
     'GOOGLE_OAUTH2_CLIENT_ID': None,
 
+    # Google reCAPTCHA
+    'GOOGLE_RECAPTCHA_SITEKEY': None,
```

Comment:
There's only one reCAPTCHA -- it's a proper name. So let's leave the "GOOGLE_" prefix off of this pair of settings, as I don't think the extra length adds anything.

Reply:
OK